schema.sql

```diff
@@ -49,3 +49,22 @@ ALTER TABLE animals
 ADD COLUMN owner_id INTEGER REFERENCES owners(id);
 
 
+-- Create a "join table" called specializations to handle the many-to-many relationship
+CREATE TABLE specializations (
+    vet_id INT,
+    species_id INT,
+    PRIMARY KEY (vet_id, species_id),
+    FOREIGN KEY (vet_id) REFERENCES vets (id),
+    FOREIGN KEY (species_id) REFERENCES species (id)
+);
+
+
+-- Create a "join table" called visits to handle the many-to-many relationship between the animals and vets tables
+CREATE TABLE visits (
+    visit_id SERIAL PRIMARY KEY,
+    vet_id INT,
+    animal_id INT,
+    visit_date DATE,
+    FOREIGN KEY (vet_id) REFERENCES vets (id),
+    FOREIGN KEY (animal_id) REFERENCES animals (id)
+);
```

Comment:
- [ ] As required for the project, kindly ensure that you have added a new table named `vets` with following information as it misses now in your `schema.sql` file.
![Screenshot from 2023-09-27 17-12-29](https://github.com/Kaiserabbas/vet_clinic/assets/83042742/5cbc28e2-a2d3-4ab7-8e55-85ced3532727)